azure-container-apps-maven-plugin/src/main/java/com/microsoft/azure/maven/containerapps/DeployMojo.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+package com.microsoft.azure.maven.containerapps;
+
+import com.fasterxml.jackson.dataformat.javaprop.JavaPropsMapper;
+import com.fasterxml.jackson.dataformat.javaprop.JavaPropsSchema;
+import com.microsoft.azure.maven.utils.MavenConfigUtils;
+import com.microsoft.azure.toolkit.lib.common.exception.AzureToolkitRuntimeException;
+import com.microsoft.azure.toolkit.lib.common.operation.AzureOperation;
+import com.microsoft.azure.toolkit.lib.containerapps.config.ContainerAppConfig;
+import com.microsoft.azure.toolkit.lib.containerapps.task.DeployContainerAppTask;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.project.MavenProject;
+import org.codehaus.plexus.util.xml.Xpp3Dom;
+
+import java.io.IOException;
+
+import static com.fasterxml.jackson.databind.DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES;
+
+/**
+ * Deploy your project to target Azure Spring app. If target app doesn't exist, it will be created.
+ */
+@Mojo(name = "deploy")
+@Slf4j
+public class DeployMojo extends AbstractMojoBase {
+
+    private static final int GET_URL_TIMEOUT = 60;
+    private static final int GET_STATUS_TIMEOUT = 180;
+    private static final String PROJECT_SKIP = "Packaging type is pom, taking no actions.";
+    private static final String PROJECT_NO_CONFIGURATION = "Configuration does not exist, taking no actions.";
+    private static final String PROJECT_NOT_SUPPORT = "`azure-container-apps:deploy` does not support maven project with " +
```

Comment:
Can we have an alias to the prefix? Say `aca`?

Reply:
add aca as global prefix in pom